packages/application-components/src/components/main-pages/custom-form-main-page/custom-form-main-page.tsx

```diff
@@ -70,31 +66,54 @@ const CustomFormMainPage = (props: CustomFormMainPageProps) => {
 
   return (
     <PageWrapper>
-      <CustomViewsSelector
-        customViewLocatorCode={props.customViewLocatorCode}
-      />
       <MainPageContainer>
-        <Spacings.Stack scale="l">
```

Comment:
Small note: I'm cool with the change, but I'm not entirely clear on what the issue is or what I might be overlooking. It seems to me that if we stick with `<Spacings.Stack scale="l">`, the margins should still be applied correctly without the need to set them individually for each part.

|  with the selector rendered | without the selector  |
|---|---|
| <img width="1061" alt="Screenshot 2023-11-30 at 10 32 03" src="https://github.com/commercetools/merchant-center-application-kit/assets/49066275/11faec48-36a8-402f-9de2-41f4a27dd4a9">  |  <img width="1047" alt="Screenshot 2023-11-30 at 10 30 40" src="https://github.com/commercetools/merchant-center-application-kit/assets/49066275/d5fee864-b205-4aed-9cec-6b4749b952f5"> |

I don't anticipate any changes—I'm good with it. Once it's updated to use the new selector styles, it's good to me.

Reply:
If I understand correctly what you suggest I guess that's something I wanted to keep but could not.
The problem is that keeping the `Spacing.Stack` component will introduce an unexpected margin then there are not custom views available. In this situation, the selector is rendered anyway with zero height because that's needed for the potential animation in case there are Custom views available. As there is a "ghost" HTML element, the `Spacing.Stack` introduces that unwanted margin.

Maybe this explanation is not good enough so I would be open for a call where I can also demonstrate the issue if you want to.

Reply:
Ah, I see now. I overlooked the transition.